source/teiid.py

```diff
@@ -95,9 +95,8 @@ def query(self, query):
         # Parse columns, fetch rows
         columns = [desc[0] for desc in cursor.description]
         rows = cursor.fetchall()
-        # Convert longs into ints and dictify the table
-        rows = [[int(item) if isinstance(item, long) else item
-                for item in row] for row in rows]
+        # dictify the table
+        rows = [[item for item in row] for row in rows]
```

Comment:
How is this change related to the `xmlrpc` driver update? Also, if I remember correctly we still want to keep python 2 support, right?